File: app/src/main/java/com/past/music/activity/CollectedActivity.java
package com.past.music.activity;

import android.os.Bundle;

import com.past.music.pastmusic.R;

public class CollectedActivity extends ToolBarActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setTitle("我喜欢");
    }

    @Override
    protected int getLayoutId() {
        return R.layout.activity_collected;
    }
}

File: app/src/main/java/com/past/music/activity/CollectedActivity.kt
package com.past.music.activity

import android.os.Bundle
import com.past.music.pastmusic.R

/**
 * Created by gaojin on 2017/11/26.
 */
class CollectedActivity : ToolBarActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        title = "我喜欢"
    }

    override fun getLayoutId(): Int {
        return R.layout.activity_collected
    }

}